src/main/java/uk/ac/cam/cl/dtg/segue/comm/EmailManager.java

```diff
@@ -255,7 +256,7 @@ public void sendCustomEmail(final RegisteredUserDTO sendingUser, final String co
         ImmutableMap<String, Object> eventDetails = new ImmutableMap.Builder<String, Object>().put("userIds", ids)
```

Comment:
I know this is not effected in this commit but are we logging everything we need in this event? I.e. do we record the admin user who did it?